src/v_video.c

```diff
@@ -858,6 +858,36 @@ void V_DrawHorizLine(int x, int y, int scrn, int width, byte color)
   }
 }
 
+void V_ShadeScreen(void)
+{
+  int y;
+  byte *dest = screens[0];
+  const int targshade = 20;
+  static int oldtic = -1;
+  static int screenshade;
+  
+  // [FG] longer than one tic ago? start a new sequence
```

Comment:
Native speakers, I know it's just a comment, but is it "more than one tic" or "longer than one tic"?

Reply:
In case you're expecting an answer from me, assuming that I am a native speaker, I'll let you know that I'm not. Regardless, I don't know either, but I think the meaning gets through anyways.

Reply:
It's "more than one tic ago". I wouldn't say "longer" is impossible in this context, but it doesn't sound natural.